data/json/item_actions.json

```diff
@@ -731,5 +731,10 @@
         "type" : "item_action",
         "id" : "ups_based_armor",
         "name" : "Activate/deactivate"
+    },
+    {
+        "type" : "item_action",
+        "id" : "CATTLEFODDER",
+        "name" : "Place"
```

Comment:
"Feed cattle" would be better - "Place" sounds like you're placing it like a trap.